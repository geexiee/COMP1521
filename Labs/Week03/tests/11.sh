./maf 0 10000000 00000000000000000000000
