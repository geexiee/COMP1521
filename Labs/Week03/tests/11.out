bits : 0 10000000 00000000000000000000000
float: 2.0000000000
